package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.39",
+  "publisher": "Aqovia",
+  "license": "MIT",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
+    "42crunch.vscode-openapi",
+    "adisreyaj.swagger-snippets",
+    "analytic-signal.preview-pdf",
+    "arjun.swagger-viewer",
+    "bierner.lit-html",
+    "corylulu.csharp-interpolated-string-converter",
+    "christian-kohler.npm-intellisense",
+    "DavidAnson.vscode-markdownlint",
+    "dbaeumer.vscode-eslint",
+    "donjayamanne.git-extension-pack",
+    "DotJoshJohnson.xml",
+    "eamodio.gitlens",
```

Comment:
Not doing any harm but I think that this one is already included as part of `donjayamanne.git-extension-pack`

Reply:
Removed this.